namespace fixes

moljac opened the pull request:
### Does this change any of the generated binding API's?

yes - namespaces

### Describe your contribution

fixed namespaces for newer versions of packages

jpobst commented:
I don't think we can change namespaces if these package have already shipped.  When we fixed the spelling of `Accessibility` in a namespace it broke the entire ecosystem for ~6 months:

https://github.com/xamarin/AndroidX/issues?q=is%3Aissue+accessibiity+is%3Aclosed

I think all we can do is live with our mistake and try to do better in the future.  😢

moljac commented:
This will cause problems with future namespace checks. Now we'll need list of "mistakes".

jpobst commented:
Let's figure out when we made each of these mistakes.

Like the Material ones have only been released for 13 days, I think we can fix that one without causing too much pain, since few people will be using it yet.

moljac commented:
I think it was in last 2 weeks. I am quite sure not before

jpobst commented:
Yep, got lucky then.  We should be able to fix this with minimal pain.

I thought the build process was checking for new namespaces.  Do we need to do additional work to ensure it fails the build if we don't fix them?

moljac commented:
> I thought the build process was checking for new namespaces. 

No. It is in `utilities.cake`. I run it by default before raising PR.

> Do we need to do additional work to ensure it fails the build if we don't fix them?

Not much. I don't write to file, but I could.

moljac commented:
This is my standard build cmd:

```
dotnet cake -t=clean && dotnet cake -t=ci && dotnet cake nuget-diff.cake && dotnet cake utilities.cake
```

jpobst commented:
It's a good check, we should add it to our CI when we get a chance to prevent me (or other contributors) from making this mistake again.  😁

jpobst reviewed (approved):
I added the NuGet version bumps so we can get these packages out before too many people rely on the ones with bad namespaces.

Good catch on this.  My bad that I did not catch these when I updated these.  😢

moljac commented:
> I added the NuGet version bumps so we can get these packages out before too many people rely on the ones with bad namespaces.

Thanks. I planned to do that too.

> Good catch on this. My bad that I did not catch these when I updated these. 😢

Hey. At least our mistakes are not lethal.

Merging, tagging and releasing so everything is out before fire.
